src/triggers/default_triggers.py

```diff
@@ -57,6 +57,12 @@
         "lso_operator": OlmOperatorsTrigger(condition="lso"),
         "cnv_operator": OlmOperatorsTrigger(condition="cnv"),
         "odf_operator": OlmOperatorsTrigger(condition="odf"),
+        "ipxe_boot": Trigger(
+            condition=("ipxe_boot", True),
+            download_image=False,
+            master_boot_devices=["hd", "network"],
+            worker_boot_devices=["hd", "network"],
+        ),
```

Comment:
Signature will be different when https://github.com/openshift/assisted-test-infra/pull/1523 will merge
```suggestion
        "ipxe_boot": Trigger(
            condition=lambda config: config.ipxe_boot == True,
            download_image=False,
            master_boot_devices=["hd", "network"],
            worker_boot_devices=["hd", "network"],
        ),
```

Reply:
Made the change, but used `is` instead of `==`